vipps-ecom-api.md

```diff
@@ -925,75 +922,22 @@ See FAQ:
 When the user confirms the purchase in Vipps, the payment status changes to `RESERVE`.
 The respective amount will be reserved for future capturing.
 
-For example:
-![Payment confirmation](images/vipps-flow-reserve.png)
-
-See the FAQ:
-* [For how long is a payment reserved?](vipps-ecom-api-faq.md#for-how-long-is-a-payment-reserved)
+For more details, see
+[Common topics: Reserve and capture](https://github.com/vippsas/vipps-developers/blob/master/common-topics/reserve-and-capture.md).
 
 ## Capture
 
 Capture is done with
 [`POST:/ecomm/v2/payments/{orderId}/capture`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/capturePaymentUsingPOST).
 
-We strongly recommend that you use the idempotency key, `X-Request-Id`, in the capture call. Then, if a capture
+We strongly recommend that you use the idempotency key,
```

Comment:
Lets just say you have to send it (yes I know its optional, but better to make people think its required)